db/migrate/20200207104825_create_books.rb

```diff
@@ -0,0 +1,15 @@
+class CreateBooks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :books do |t|
+      t.string :title, null: false
+      t.string :author, null: false
+      t.text :description
+      t.string :isbn10, limit: 10
+      t.string :isbn13, limit: 13
+      t.date :year
```

Comment:
I think we should rename this column to 'release_date' or something similar. `year` is a bit misleading here, because the value won't actually be just the year, it's a full date.

Reply:
Agree, I changed it.